scripts/cc-cli/src/commands/receive.ts

```diff
@@ -16,7 +17,8 @@ export function makeReceiveCommand() {
   return cmd;
 }
 
-function receiveAction(options: OptionValues) {
+async function receiveAction(options: OptionValues) {
+  const { api } = await newApi();
```

Comment:
That's an unused variable. Do we need it for anything ? Not sure I understand.

Reply:
Ahh, there is some misunderstanding here. Let me ask again. 

1) Why do we need to wait at all? What for? 

IMO if the wait is indeed necessary using `newApi()` is preferable for consistency, just prefix the variable with underscore to satisfy eslint. And maybe add a short comment explaining what are we waiting for. That's what wasn't clear to me.

Reply:
Oh, the wait is necessary because the KeyRing uses a WASM library for `sr25519` addresses and we have to wait for it to initialize. https://polkadot.js.org/docs/ui-keyring/start/init/

Now, I would rather not use the `newApi()` cause that means that it depends on a running node to just generate an address.